### ISSUE TITLE
it is not showing the related files/content

### ISSUE THREAD

**Issue by pmappway** [DouglasCAyers/sfdc-related-files-lightning#31]
I have followed your instructions but the related files component remains empty (see attachment)
![related files](https://user-images.githubusercontent.com/51917791/59610136-0e33da80-9119-11e9-8607-21fa0d2ed6f6.jpg)

**Comment by pmappway** [comment]
@douglascayers do you need more information on this? Any updates? I could directly show it via screen-sharing. Thanks for your reply

**Comment by douglascayers** [comment]
Thanks @pmappway, would you please also share screenshot of the component's configuration in App Builder?

And, I'm curious what errors (if any) are being printed to the browser's console. In most modern browsers, you can right-click anywhere on the page and choose **Inspect**.

![image](https://user-images.githubusercontent.com/4142577/59971454-ffe32580-9541-11e9-8e62-6cbfe7aa8e57.png)

Some developer tools will open up on the page, look for a tab named **Console** and click it.

![image](https://user-images.githubusercontent.com/4142577/59971449-f5289080-9541-11e9-995b-37305dccbbc0.png)

Share a screenshot of any output, particularly any red messages.

Thanks

**Comment by pmappway** [comment]
Hello Douglas

See below all the screenshots. I’m more than happy to jump on a quick screensharing session if that would help solving the issue quickly.
Thanks & have a nice day,
Philipp


Here the screenshot of the app builder:
[cid:image001.jpg@01D52A76.6D75C580]

Now there is even an apex error message as it is loading very slowy and the related files component is not really properly displaying:
[cid:image002.jpg@01D52A76.6D75C580]

Screenshot of the inspect console (I did it 2 times):

[cid:image007.png@01D52A76.6D6598E0]

[cid:image008.png@01D52A76.6D6598E0]

Greetings,
Philipp
+41 (0) 43 204 09 55


How has SIX taken its global client services to the next level? Read our ZHAW case study in English or in German to find out.
Connect with us: appway.com | twitter | linkedin | newsletter
This email and any attachments are confidential and intended solely for the person and/or company to whom they are addressed. If you have received this email in error please notify the sender and please do not disseminate or distribute this email. If you are not the intended recipient, disclosing, copying, distributing or taking any action on the contents of this email is strictly prohibited.

**Comment by pmappway** [comment]
@douglascayers  any update on this? I'd suggest to jump on a call to resolve this quickly. I can adjust towards your availabilities pls let me know, thanks!

**Comment by pmappway** [comment]
hello @douglascayers  I would really appreciate your effort to come back to me to have a look at my issue. many thanks

**Comment by pmappway** [comment]
@douglascayers  any updates on this? do I miss anything or why is the related files navigation pane not visible? the whole red marked part is not displaying (but there are actually numerous files existing on that record and related-records..)
![related filed navigation list not visible](https://user-images.githubusercontent.com/51917791/63498696-b625a580-c4c6-11e9-81b1-81f19283d939.jpg)

**Comment by jsferrebee** [comment]
@pmappway  - don't know if you got any resolution on this, but I had the exact same problem.  I debugged and was getting a 'Too Many Call-outs' from the CF_ChildFilesController.cls.  I could recreate by just running the CF_ChildFilesControllerTest.cls.  I guessed it was because our org had too many custom fields and that class makes a call-out to the Tooling API for any custom field in the queryCustomFieldMetadate method.  Filtering to only specific custom fields seems to correct the issue.
